index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="navigation">
+            <a class="logo" href="#">Hami</a>
+            <img class="humburger" src="images/Union.svg" alt="">
+        </nav>
+    </header>
+    <main>
+        <h1 class="hero">I’m Hamdullah
+            Glad to see you!
+        </h1>
```

Comment:
- [ ] kindly consider taking a `section` tag under `main` tag and putting everything in the headline section inside it. and that is due to this :  
 -  It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.